tiledb/sm/query/readers/result_tile.cc

```diff
@@ -335,6 +335,28 @@ uint64_t ResultTile::timestamp(uint64_t pos) {
   return tile.data_as<uint64_t>()[pos];
 }
 
+template <typename LabelType>
+LabelType ResultTile::attribute_value(
+    const std::string& label_name, const uint64_t pos) {
+  const auto label_data =
+      tile_tuple(label_name)->fixed_tile().template data_as<LabelType>();
+  return label_data[pos];
+}
+
+template <>
+std::string_view ResultTile::attribute_value<std::string_view>(
+    const std::string& label_name, const uint64_t pos) {
+  auto tuple = tile_tuple(label_name);
+  auto offsets_data = tuple->fixed_tile().template data_as<uint64_t>();
+  auto& var_tile = tuple->var_tile();
+  auto offset = offsets_data[pos];
+
+  auto size = static_cast<size_t>(pos) == cell_num() - 1 ?
+                  var_tile.size() - offset :
+                  offsets_data[pos + 1] - offset;
+  return std::string_view(&var_tile.template data_as<char>()[offset], size);
```

Comment:
Are the 'template' qualifiers here and two other locations above required for successful compilation on some platform (mac/clang?)?
Seems to compile ok for me without the 'template' on MSVC 19,22, gcc 9.4.0, (mingw) gcc 10.3...